docs/en/operation/Remote-Logging.md

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
did you edit the wording in this file from where it was copied? if not then i won't look too closely at this

Reply:
most of the text is just copy & paste. I would not suggest spending time on this doc, very few people use this feature (I am even for removing or deprecating this feature)

Reply:
got it